docs/manage/manage/oauth.md

```diff
@@ -48,9 +44,7 @@ Successfully set support email for organization "<org-id>" to "support@logoipsum
 
 {{< table >}}
 {{% tablestep link="/dev/tools/cli/#organizations" %}}
-**1. Enable authentication**
-
-Enable the authentication service for your organization:
+**1. Enable the authentication service** for your organization:
```

Comment:
Nice. Realizing this is somewhat similar to what I did here https://docs.viam.com/operate/get-started/other-hardware/#implement-the-component-api. I was looking at this page^ in the circle number CSS PR and I think the bold font looks a bit strange with the big numbers, so may revisit in future if you agree but looks good for now.